### PR TITLE
Add instrumentation to Clickhouse 

### PR DESCRIPTION
**Pull request by thiagohora**
## Issues

Resolves #

### CODE REVIEW

File: apps/opik-backend/src/main/java/com/comet/opik/utils/ClasspathChecker.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.comet.opik.utils;
+
+public class ClasspathChecker {
+
+    public static boolean isClassPresent(String className) {
```
Comment (Nimrod007) [inline apps/opik-backend/src/main/java/com/comet/opik/utils/ClasspathChecker.java]: @thiagohora what is this for ?

**Review by Nimrod007 — CHANGES_REQUESTED** [review]

File: apps/opik-backend/src/main/java/com/comet/opik/utils/ClasspathChecker.java (hunk 1 above)
Comment (thiagohora) [reply apps/opik-backend/src/main/java/com/comet/opik/utils/ClasspathChecker.java]: Removed

**Review by Nimrod007 — APPROVED** [review]

File: apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java [hunk 2]
```diff
@@ -612,6 +626,9 @@ private Flux<? extends Result> bindAndDelete(List<UUID> ids, Statement statement
 
     @Override
     public Mono<DatasetItemPage> getItems(@NonNull UUID datasetId, int page, int size) {
+
+        Segment segment = startSegment("dataset_items", "Clickhouse", "select_dataset_items_page");
```
Comment (andrescrz) [inline apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java]: Where is this segment ending?

**Review by andrescrz — APPROVED** [review]

File: apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java (hunk 2 above)
Comment (thiagohora) [reply apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java]: Good catch

**Review by andrescrz — APPROVED** [review]

**Pull request merged**